www/static/admin/config.js

```diff
@@ -31,7 +31,7 @@ const defaultFields = {
     {
       label: "Status",
       name: "status",
-      widget: "boolean",
+      widget: "string",
```

Comment:
Noticed that our `WIP` badge in palette docs was incorrect (had updated, but didn't switch from a boolean to a string, which allows us to pass different things).